src/components/Alert/Alert.stories.tsx

```diff
@@ -38,6 +38,12 @@ storiesOf('Alert', module)
 			</div>
 		</React.Fragment>
 	))
+	.add('Alerts html multi line', () => (
+		<Alert
+			message={'Info alert message<br/>Info alert message<br/>Info alert message'}
```

Comment:
DIt moet niet als string, je kan gwn dit doen:
```
message={(
    <Fragment>Info alert message<br/>Info alert message<br/>Info alert message</Fragment>
)}`

Reply:
hmm, ja dat is beter, closing...